google-cloud-language/MIGRATING.md

```diff
@@ -81,6 +81,16 @@ Google::Cloud::Language::V1::LanguageService::Client.configure do |config|
 end
 ```
 
+You can also set certain configuration defaults for all Language versions
+globally:
```

Comment:
Why not lead with the broadest scope first? Then the progression would be consistently general -> specific.

Reply:
The ordering is intentional. It leads with the most straightforward and common cases, and moves toward less common cases.